galleon-pack/src/main/resources/layers/standalone/bean-validation/layer-spec.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="cdi">
```

Comment:
This name should probably be 'bean-validation'?

Reply:
@kabir Fixed; thanks!